uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -102,29 +101,26 @@ define([
      * @return {Boolean}
      */
     function checkSubmitConditions () {
-      return !!vm.request.from_date && !!vm.request.to_date &&
-        !!vm.request.toil_duration && !!vm.request.toil_to_accrue;
+      return !!parentCtrl.request.from_date && !!parentCtrl.request.to_date &&
+        !!parentCtrl.request.toil_duration && !!parentCtrl.request.toil_to_accrue;
     }
 
     /**
      * Extends parent method. Fires calculation of expiry date when the
```

Comment:
> Extends parent method

It doesn't, it's simply an additional method added to the parent controller, there is no `changeInNoOfDaysExtended` on the parent controller being extended here

Reply:
Done.